client/ratings_reviews/components/ReviewForm.jsx

```diff
@@ -1,9 +1,36 @@
 import React from 'react';
+import ReviewFormModal from './ReviewFormModal.jsx';
 
-const ReviewForm = (props) => (
-  <div>
-    reviewform
-  </div>
-);
+class ReviewForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      show: false
+    };
+    this.showModal = this.showModal.bind(this);
```

Comment:
If you want to simplify showModal and closeModal I think you can combine them into a single function (like toggleModal) - I did something like this for my modal if you want to check it out!

Reply:
Hi @jabrake, thank you for pointing this out! I tried combining them into one before but it didn't work. I will take a look a your code later to see if I can make changes to this. Thank you!